libs/commons-database/src/main/kotlin/gov/cdc/ocio/database/cosmos/CosmosCollection.kt

```diff
@@ -0,0 +1,140 @@
+package gov.cdc.ocio.database.cosmos
+
+import com.azure.cosmos.CosmosContainer
+import com.azure.cosmos.models.CosmosItemRequestOptions
+import com.azure.cosmos.models.CosmosQueryRequestOptions
+import com.azure.cosmos.models.PartitionKey
+import gov.cdc.ocio.database.persistence.Collection
+import io.netty.handler.codec.http.HttpResponseStatus
+import mu.KotlinLogging
+
+
+/**
+ * Cosmos Collection implementation
+ *
+ * @property cosmosContainer CosmosContainer?
+ * @constructor
+ */
+class CosmosCollection(
+    containerName: String,
```

Comment:
`containerName` parameter is missing from Kdoc comment

Reply:
Fixed